cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -6,20 +6,21 @@
 
 #include <Camera/CameraComponent.h>
 #include <Components/InputComponent.h>
+#include <GameFramework/PlayerInput.h>
 
 #include "CoreUtils/Config.h"
 #include "CoreUtils/Unreal.h"
 #include "UrdfBot/UrdfParser.h"
 #include "UrdfBot/UrdfRobotComponent.h"
 
-AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer) : APawn(object_initializer)
+AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer)
+    : APawn(object_initializer)
 {
     UrdfRobotDesc robot_desc = UrdfParser::parse(Unreal::toStdString(FPaths::Combine(
         Unreal::toFString(Config::get<std::string>("URDFBOT.URDFBOT_PAWN.URDF_DIR")),
         Unreal::toFString(Config::get<std::string>("URDFBOT.URDFBOT_PAWN.URDF_FILE")))));
 
     robot_component_ = CreateDefaultSubobject<UUrdfRobotComponent>(Unreal::toFName("AUrdfBotPawn::robot_component_::" + robot_desc.name_));
-    robot_component_->initializeComponent(&robot_desc);
     robot_component_->createChildComponents(&robot_desc);
 
     RootComponent = robot_component_;
```

Comment:
Use Unreal::toFName("AUrdfBotPawn::camera_component_") instead of TEXT("AOpenBotPawn::camera_component_")

Reply:
fixed